sdk/storage/azure-storage-file-share/azure/storage/fileshare/_lease.py

```diff
@@ -145,22 +190,42 @@ def change(self, proposed_lease_id, **kwargs):
         self.last_modified = response.get('last_modified')   # type: datetime
 
     @distributed_trace
-    def break_lease(self, **kwargs):
+    def break_lease(self, lease_break_period=None, **kwargs):
```

Comment:
This parameter should also be moved into kwargs - especially as it only applies to Shares

Reply:
> This would be inconsistent to `\blob\_lease.py`, do we not want to maintain the same method definition/pattern as `\blob\_lease.py`?

this comment applies here as well

Reply:
In this case it should be in kwargs.
It's still consistent with blobs in that the parameter is named the same - so from a user perspective it's consistent.